Amazon.QLDB.Driver/utils/QldbHash.cs

```diff
@@ -30,6 +30,7 @@ namespace Amazon.QLDB.Driver
     internal class QldbHash
     {
         private const int HashSize = 32;
+        private static readonly HashAlgorithm Sha256Algorithm = SHA256.Create();
```

Comment:
Note that you'll have to use this below on line 115-116, which is still using the local string-based version.

```cs
HashAlgorithm hashAlgorithm = HashAlgorithm.Create("SHA256");
return new QldbHash(hashAlgorithm.ComputeHash(concatenated));
```